<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paySelect</class>
 <widget class="QWidget" name="paySelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="previousPage_Button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>1820</y>
     <width>281</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="payPage_Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>1590</y>
     <width>751</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="orderSmall_Button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>1160</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="orderBig_Button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>1160</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mainPage_Button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>1740</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>1386</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Proxima Nova;
background-image: url(:/light/background.png);
font-style: normal;
font-weight: normal;
font-size: 24px;
line-height: 29px;

color: #58595B;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="totalPriceLabel">
   <property name="geometry">
    <rect>
     <x>806</x>
     <y>1498</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Proxima Nova;
background-image: url(:/light/background.png);
font-style: normal;
font-weight: bold;
font-size: 36px;
line-height: 44px;

color: #58595B;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
